pkgs/applications/misc/qownnotes/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchgit, qt5 }:
```

Comment:
You are using `qt5.callPackage`, so instead of importing `qt5` here, you must import the specific submodules you use. This is so that your package sees a consistent set of dependencies.